app.js

```diff
@@ -1,31 +1,30 @@
-// const express = require('express');
-// const routes = require("./routes")
-// const fs = require("fs");
+const express = require('express');
+const routes = require("./routes")
```

Comment:
To stay consistent, please make sure, that there no missing `;` in the codebase
```suggestion
const routes = require("./routes");
```

Reply:
done